src/datumaro/components/dataset_storage.py

```diff
@@ -643,7 +705,15 @@ def stacked_transform(self) -> IDataset:
         return transform
 
     def __iter__(self) -> Iterator[DatasetItem]:
-        yield from self.stacked_transform
+        # yield from self.stacked_transform
```

Comment:
This line can be deleted.

Reply:
fixed at 0a313ca5d4b9e08208b20a226840d026456c1655